README.md

```diff
@@ -50,10 +50,11 @@ Returns source map converter from given `filename` by parsing `//# sourceMapping
 `filename` must point to a file that is found inside the `mapFileDir`. Most tools store this file right next to the
 generated file, i.e. the one containing the source map.
 
-### fromSource(source)
+### fromSource(source[, largeSource])
 
-Finds last sourcemap comment in file and returns source map converter or returns null if no source map comment was
-found.
+Finds last sourcemap comment in file and returns source map converter or returns null if no source map comment was found.
+
+`largeSource` is optional. If set to `true`, a faster, but potentially less accurate algorithm is used. This is useful if you're running into "call stack size exceeded" errors.
```

Comment:
Maybe we should point out here that it's an algorithm that **uses no regex** and therefore is faster, avoids `call stack size exceeded` but is may match things that aren't proper source maps.